README.md

```diff
@@ -142,3 +142,11 @@ BatchPush.fetchNewNotifications('theUserId', 'authKey')
   })
   .catch(e => console.warn('BatchPush error', e));
 ```
+
+### Last known push token
+
```

Comment:
Please add a warning that this should only be used for debugging - the correct way way in production would be to [use a listener](https://batch.com/doc/android/advanced/getting-registration.html#_listening-to-the-broadcast)

Reply:
It is my understanding that Batch already implements a listener and stores the push token when it changes. Why add one more listener on top of that?

Reply:
I'm only referencing what states in the doc :) It looks like we shouldn't rely too much on this value.

> This method will return null if no registration identifier has been fetched or if Batch isn't started, meaning that you have to call this method after Batch.onStart(). It is useful if you want to show the token in your UI, for example in a debug view.

Reply:
This (annoying) limitation is exactly why we introduced the listener down the road: users had to work around the token not being available with something like setTimeout.

The listener also gives you the token provider (GCM/FCM, and possibly more in the future)

My 2cts is that both can be used: they're not meant for the same use cases.